src/fastcs_pandablocks/panda/panda_controller.py

```diff
@@ -40,26 +48,70 @@ async def connect(self) -> None:
 
     async def initialise(self) -> None:
         await self.connect()
+
         for block_name, block in self._blocks.controllers():
-            self.register_sub_controller(block_name, block)
+            # Numerically named controllers are registered to
+            # alphabetically named ControllerVectors, so only
+            # alalphabetically named controllers
+            # should be registed to top level Controller
+            if str(block_name).isalpha():
+                self.add_sub_controller(block_name.lower(), block)
+
+    async def update_field_value(self, panda_name: PandaName, value: str | list[str]):
+        """Update a panda field with either a single value or a list of words."""
 
-    async def update_field_value(self, panda_name: PandaName, value: str):
         attribute = self._blocks.get_attribute(panda_name)
-        attribute_value = panda_value_to_attribute_value(attribute.datatype, value)
+        assert isinstance(attribute, AttrR)
+        if attribute is None:
+            LOGGER.error(f"Couldn't find panda field for {panda_name}.")
+            return
+
+        try:
+            attribute_value = self._coerce_value_to_panda_type(attribute, value)
+        except ValueError as e:
+            LOGGER.error(str(e))
```

Comment:
```suggestion
            logger.opt(exception=True).error("Coerce failed")
```
This will print the exception nicely